### DISCUSSION TITLE
Malt logo

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#388] [Q&A]
Is this the official malt logo that appears on the malt3d website?
If so should we use this in the malt UI? For example for the malt node tree editor
If not, is there already some kind of logo or idea? 

![image](https://user-images.githubusercontent.com/88381426/187798827-595610fe-4039-4536-8f5d-67f9fab9e62c.jpeg)

**Comment by pragma37** [comment]
LOL, nope. Malt doesn't have a logo (yet?).
IIRC, mkdocs required to put an icon, so I just googled for a cc0 "3d" icon.

> If so should we use this in the malt UI? For example for the malt node tree editor

Is it actually possible to do so?

**Comment by Kolupsy** [comment]
> Is it actually possible to do so?

I know you can make your own icons in general. I never tried to use a custom icon for a custom node tree though...

**Reply by pragma37** [reply]
Oh, that's nice. I didn't know that was possible!
